src/Sundials.jl

```diff
@@ -16,14 +31,9 @@ function __init__()
     global warnida  = union(warnlist, Set((:dtmin,)))
 end
 
-const depsfile = joinpath(dirname(dirname(@__FILE__)),"deps","deps.jl")
-if isfile(depsfile)
-    include(depsfile)
-else
-    error("Sundials is not properly installed. Please run Pkg.build(\"Sundials\")")
-end
+using Sundials_jll
 
-export solve, SundialsODEAlgorithm, SundialsDAEAlgorithm, ARKODE, CVODE_BDF, CVODE_Adams, IDA
+export solve, SundialsODEAlgorithm, SundialsDAEAlgorithm, ARKStep, CVODE_BDF, CVODE_Adams, IDA
```

Comment:
I don't think we need to breaking change rename the algorithm.

Reply:
OK, I will revert the change. From the Sundials documentation, it seems that there are significant changes to the problem formulation too. Updating this will take a while

Reply:
fixed via 8c9f360